### PR TITLE
fix: [PAYM-2345] fixed navbar

### PR DESCRIPTION
**Pull request by MekhrangizY**
**PR description:**

**Implemented:**

**Checklist:**

- [x] I ran this code locally
- [ ] I wrote the necessary tests
- [x] My code follows the [style guidelines](http://bit.ly/sd-web-style-guide)
- [x] I followed the [instructions](http://bit.ly/sd-web-pr) to create a pull request

**JIRA card:**

**Should know about:**

### CODE REVIEW

File: packages/lab/src/navbar/NavbarAccordion.tsx [hunk 1]
```diff
@@ -33,10 +33,11 @@ const NavbarAccordionRoot = styled(Accordion)`
   text-decoration: none;
   outline: none;
   cursor: pointer;
-  background-color: #1b2638;
+  background-color: ${({ theme }) =>
+    theme.palette.type === 'light' ? '#1b2638' : ColorDark.White};
```
Comment (doniyor2109) [inline packages/lab/src/navbar/NavbarAccordion.tsx:37]: Lets chack for dark first

Comment (MekhrangizY) [reply packages/lab/src/navbar/NavbarAccordion.tsx:37]: done

File: packages/lab/src/navbar/NavbarList.tsx [hunk 2]
```diff
@@ -37,7 +42,8 @@ const Wrapper = styled.div<{ isMobile: boolean }>`
 
   overflow: hidden;
   padding-top: 16px;
-  background: #1b2638;
+  background: ${({ theme }) =>
+    theme.palette.type === 'light' ? '#1b2638' : ColorDark.White};
```
Comment (doniyor2109) [inline packages/lab/src/navbar/NavbarList.tsx:46]: Lets check dark first

Comment (MekhrangizY) [reply packages/lab/src/navbar/NavbarList.tsx]: done

**Review by doniyor2109 — APPROVED** [review]

**Pull request merged**